### PR TITLE
Redefining packages and interfaces of service,credential,storage,monitor

### PR DESCRIPTION
**Pull request by baixinsui**
@iskey Please review it.

### CODE REVIEW

File: modules/orchestrator/src/main/java/org/eclipse/xpanse/orchestrator/storage/DeployResourceStorage.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+ * Interface for persist of DeployResource.
+ */
+public interface DeployResourceStorage {
+
+    void deleteByDeployServiceId(UUID id);
+
+    /**
+     * Interface to be implemented by register service database.
+     */
+    interface RegisterServiceStorage {
```
Comment (swaroopar) [inline modules/orchestrator/src/main/java/org/eclipse/xpanse/orchestrator/storage/DeployResourceStorage.java:24]: Do we really need such a nested interface?

**Review by swaroopar — CHANGES_REQUESTED** [review]
package separation looks good. 
Only question about nested interfaces

**Review by swaroopar — APPROVED** [review]
@baixinsui looks fine for me. If Yichao has no further comments, we can merge it.

**Review by iskey — APPROVED** [review]
looks good

**Pull request merged**